source/includes/aggregation/group-total.js

```diff
@@ -0,0 +1,134 @@
+const { MongoClient } = require("mongodb");
+
+const uri = "<connection string>";
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    const aggDB = client.db("agg_tutorials_db");
+
+    // start-coll
+    const collName1 = await aggDB.collection("orders");
+    // end-coll
+
+    const sampleData1 = [
+      // start-orders
+      {
+        customer_id: "elise_smith@myemail.com",
+        orderdate: new Date("2020-05-30T08:35:52Z"),
+        value: 231.43,
+      },
+      {
+        customer_id: "elise_smith@myemail.com",
+        orderdate: new Date("2020-01-13T09:32:07Z"),
+        value: 99.99,
+      },
+      {
+        customer_id: "oranieri@warmmail.com",
+        orderdate: new Date("2020-01-01T08:25:37Z"),
+        value: 63.13,
+      },
+      {
+        customer_id: "tj@wheresmyemail.com",
+        orderdate: new Date("2019-05-28T19:13:32Z"),
+        value: 2.01,
+      },
+      {
+        customer_id: "tj@wheresmyemail.com",
+        orderdate: new Date("2020-11-23T22:56:53Z"),
+        value: 187.99,
+      },
+      {
+        customer_id: "tj@wheresmyemail.com",
+        orderdate: new Date("2020-08-18T23:04:48Z"),
+        value: 4.59,
+      },
+      {
+        customer_id: "elise_smith@myemail.com",
+        orderdate: new Date("2020-12-26T08:55:46Z"),
+        value: 48.5,
+      },
+      {
+        customer_id: "tj@wheresmyemail.com",
+        orderdate: new Date("2021-02-29T07:49:32Z"),
+        value: 1024.89,
+      },
+      {
+        customer_id: "elise_smith@myemail.com",
+        orderdate: new Date("2020-10-03T13:49:44Z"),
+        value: 102.24,
+      },
+      // end-orders
+    ];
+
+    await collName1.deleteMany({});
+    await collName1.insertMany(sampleData1);
```

Comment:
Suggestion:

I think this should match the spacing of the template app as it could otherwise suggest other differences.

```suggestion
    await collName1.deleteMany({});
    
    await collName1.insertMany(sampleData1);
```